dCommon/dCommonVars.h

```diff
@@ -469,6 +469,31 @@ enum eRebuildState : uint32_t {
 	REBUILD_INCOMPLETE
 };
 
+enum eCoinSourceType : int32_t {
```

Comment:
Note that this enum is used not only for coins, but also for items (e.g. for achievements it delays the reward pickup animation until after the achievement complete animation has completed), so this should be called `eLootSourceType`. This is also what the game calls it internally. Same goes for the prefixes of the enum variants.

Reply:
Ah gotcha.  [I have updated the enum name and prefixes](https://github.com/DarkflameUniverse/DarkflameServer/pull/291/commits/6ff5ae78aa0f27b8e13bb65eea75085fe6c6669c) to be correct.